nautobot/dcim/filters/__init__.py

```diff
@@ -961,6 +965,11 @@ class DeviceFilterSet(
         field_name="interfaces",
         label="Has interfaces",
     )
+    interfaces = NaturalKeyOrPKMultipleChoiceFilter(
```

Comment:
I see `interfaces` declared in `Meta.fields` - is this just turning the default model choice filter into a NaturalKeyOrPKMultipleChoiceFilter?

Good to document as a change fragment in any case.